src/CategoryDescription.css

```diff
@@ -0,0 +1,69 @@
+@media only screen and (max-width: 767px) {
+  .category-container {
```

Comment:
grid-template-area might also be something to look into, it is very useful for declaring where everything goes instead of these ratios that can still raise questions. just something to note